api/paidAction/index.js

```diff
@@ -30,9 +30,9 @@ export const paidActions = {
   DONATE
 }
 
-export default async function performPaidAction (actionType, args, context) {
+export default async function performPaidAction (actionType, args, incomingContext) {
```

Comment:
to make this prettier we could just use  { ...context }, like:
```
export default async function performPaidAction (actionType, args, { ...context }) {
```
on every function that requires the context, so they get a shallow copy each